app/views/claims/support/schools/mentors/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :page_title, mentor_form.errors.any? ? t(".page_title_with_error", school_name: @school.name) : t(".page_title", school_name: @school.name) %>
+<% render "claims/support/primary_navigation", current: :organisations %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_support_school_mentors_path(@school)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <%= form_with(model: mentor_form, url: check_claims_support_school_mentors_path, method: "get") do |f| %>
+    <%= f.govuk_error_summary %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-form-group">
+          <%= f.govuk_text_field :trn, width: "two-thirds", label: -> do %>
+            <span class="govuk-caption-l"><%= t(".add_mentor_with_school_name", school_name: @school.name) %></span>
+            <h2 class="govuk-heading-l"><%= t(".enter_a_trn") %></h2>
+          <% end %>
+        </div>
+
+        <%= govuk_details(summary_text: t(".help_with_the_trn")) do %>
+          <p>
+            <%= t(".if_you_dont_have_a_trn") %>
+            <%= govuk_link_to(
+                  t(".trn_guidance"),
+                  "https://www.gov.uk/guidance/teacher-reference-number-trn",
+                  new_tab: true,
+                  no_visited_state: true,
+                ) %>
+            <%= t(".to_find_a_list_trn") %>
+          </p>
```

Comment:
Instead of splitting this into 3 distinct pieces of text, you can use a single key and pass through the `a` tag.

Although not immediately applicable, this is a better general practice, as when working with other languages, they may not order words the same way as English.

```suggestion
          <p>
            <%= t(".guidance", guidance_link: govuk_link_to(t(".guidance_link_text", "https://www.gov.uk/guidance/teacher-reference-number-trn", new_tab: true, no_visited_state: true)) %>
          </p>
```

```yaml
en:
  ...:
    guidance: If you don’t have a TRN, read the %{guidance_link} to find a lost TRN, or apply for one.
    guidance_link_text: Teacher reference number (TRN) guidance
```